File: problemas/ad-hoc/1397/1397.php
<?php
while(true){
  $n = trim(fgets(STDIN));

  if($n == 0){
    break;
  }
  
  $jogador1 = 0;
  $jogador2 = 0;
  for($i = 0; $i < $n; $i++){
    $entrada = array_map('intval', explode(" ", trim(fgets(STDIN))));
    $a = $entrada[0];
    $b = $entrada[1];

    if($a > $b){
      $jogador1 += 1;
    } else if($a < $b) {
      $jogador2 += 1;
    }
  }
  
  echo $jogador1 . ' ' . $jogador2 . PHP_EOL;
}
?>

File: problemas/ad-hoc/1397/1397.r
input <- file('stdin', 'r')

n <- as.integer(readLines(input, n = 1))
while (n > 0) {
  jogador1 <- 0
  jogador2 <- 0
  for(i in 1:n) {
    entrada <- as.integer(strsplit(readLines(input, n = 1), ' ')[[1]])
    a <- entrada[1]
    b <- entrada[2]

    if(a > b) {
      jogador1 <- jogador1 + 1
    } else if(a < b) {
      jogador2 <- jogador2 + 1
    }
  }

  write(paste(jogador1, jogador2), '')

  n <- as.integer(readLines(input, n = 1))
}
